index.html

```diff
@@ -106,20 +104,21 @@
       </div>
     </my-app>
 
-    <script src="/bower_components/babel-polyfill/browser-polyfill.js"></script>
+    <script src="/bower_components/babel-polyfill/browser-polyfill.js" async></script>
     <script>
-      // Global Constants
-      window.APP_NAME = 'My App';
-      window.BASE_URL = '/';
```

Comment:
Removing this causes issues with the locales loading script, I added `|| '/'` to mine to fix it for Citrix.